src/components/reorderer/js/Reorderer.js

```diff
@@ -508,7 +508,7 @@ https://github.com/fluid-project/infusion/raw/main/Infusion-LICENSE.txt
         var handleClick = function (evt) {
             var handle = fluid.unwrap(thatReorderer.dom.fastLocate("grabHandle", this));
             if (fluid.dom.isContainer(handle, evt.target)) {
-                $(this).focus();
+                $(this).trigger("focus");
```

Comment:
There are now several inconsistent means of updating focus in this file - 
i) $(el).focus();
ii) $(el).trigger("focus")
iii) fluid.focus(el)
we should normalise these so that we only use one style - presumably iii). 
See notes on line 594 below

Reply:
I don't think case i) occurs. What you might have seen is an instance of calling focus on a regular DOM node. At any rate, I'll explore converting all of these to using `fluid.focus`.